test/scripts/start-services.sh

```diff
@@ -51,6 +51,28 @@ start_geth() {
         > "$output_dir/geth.log" 2>&1 &
 }
 
+start_geth_for_beacon_node() {
```

Comment:
This can be merged with `start_geth` in a follow-up PR. The reason why I added a new function for this is because the Lodestar-Geth combo requires a JWT secret, and I wasn't sure if that would break something if I added a `authrpc` parameter to our local geth setup.